test/integration/scheduled_stop_test.go

```diff
@@ -84,6 +87,8 @@ func TestScheduledStopUnix(t *testing.T) {
 
 	// schedule a stop for 5 min from now and make sure PID is created
 	stopMinikube(ctx, t, profile, []string{"--schedule", "5m"})
+	// make sure timeToStop is present in status
+	ensureMinikubeStatus(ctx, t, profile, "TimeToStop", "3m")
```

Comment:
I see that you had to change to using  `strings.Contains` in the `ensureMinikubeStatus` function, since we can't know what the exact value of `TimeToStop` will be. Instead of doing a contains, which could potentially fail, I would just check that TimeToStop is a valid duration < 5m.

Then we can also revert back to making sure `got == wantStatus` in the `ensureMinikubeStatus` function

Reply:
So, can we pull this check to a seperate function or modify the comparison based on the `key` value to the `ensureMinikubeStatus` itself ?.

Reply:
Let's do a separate function